src/server/common/test-helpers/csrf.js

```diff
@@ -79,7 +79,8 @@ export const withCsrfProtection = (injectionOptions, headers = {}) => {
     headers: {
       ...(injectionOptions.headers ?? {}),
       ...headers,
-      Cookie: ['crumb=csrf-value', headers?.Cookie].filter((v) => v).join(';')
+      Cookie: ['crumb=csrf-value', headers?.Cookie].filter((v) => v).join(';'),
+      Referer: headers.Referer ?? ''
```

Comment:
Is this necessary - since if we include it in `headers`, it'll be available here? 

If this is breaking one of our tests (e.g. for taskListGetController), then I think that implies that the controller is not robust enough for a non-essential header being missing rather than we should make sure it's always there